components/LinkToSubject.vue

```diff
@@ -16,15 +16,17 @@
 </template>
 
 <script setup lang="ts">
-import { RiDatabase2Line, RiLineChartLine } from '@remixicon/vue'
+import { RiDatabase2Line, RiLineChartLine, RiRobot2Line } from '@remixicon/vue'
+import type { LinkToSubject } from '~/types/types'
 
 const props = defineProps<{
-  type: 'Dataset' | 'Reuse'
-  subject: { title: string, page: string }
+  type: 'Dataservice' | 'Dataset' | 'Reuse'
```

Comment:
There is a problem with the link to the dataservice
![image](https://github.com/user-attachments/assets/a6e272a1-5810-4092-9a19-3e729eeb362b)